private/buf/cmd/buf/command/dep/depgraph/depgraph.go

```diff
@@ -143,5 +143,5 @@ func moduleToString(module bufmodule.Module) string {
 		}
 		return moduleFullName.String()
 	}
-	return module.OpaqueID()
+	return module.Description()
```

Comment:
Output:
```
buf dep graph
digraph {

  "buf.build/osun/separate" -> "path: &#34;foo&#34;, excludes: &#34;foo/bar&#34;"
  "path: &#34;foo&#34;, excludes: &#34;foo/bar&#34;" -> "path: &#34;foo&#34;, includes: &#34;foo/bar/baz&#34;"
  "path: &#34;foo&#34;, includes: &#34;foo/bar/baz&#34;" -> "path: &#34;foo&#34;, includes: &#34;foo/bar&#34;, excludes: &#34;foo/bar/baz&#34;"

}
```
This output is [escaped](https://github.com/bufbuild/buf/blob/385701964658963968829a8a8a83651ef813fdf7/private/pkg/dag/graph.go#L507). Perhaps we should update the definition of `Module.Description` not to quote paths.

Reply:
This is a breaking change and isn't desirable - definitely revert this

Reply:
Reverted in [9f854e6](https://github.com/bufbuild/buf/pull/3271/commits/9f854e67feb9d3732a72782e29aa0babbeacbb61)